### PR TITLE
Add an optional callback for any edit

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,11 +36,16 @@ integration and I’ll mention it here).
 ``` javascript
 var Edited = require(‘edited’)
 
-var cb = function() {
-    // register undo point, autosave, etc
+var onSensible = function () {
```
Comment (winhamwr) [inline README.md]: It's not clear from this documentation what the difference is between `onSensible` and `onAny`. What makes it `sensible`?

**Comment by winhamwr** [comment]
Other than this missing documentation on what makes the events sensible (word boundaries, etc), this looks good to me.

File: README.md (hunk 1 above)
Comment (mightyiam) [inline README.md]: @winhamwr merging and taking care of documentation in master.

**Comment by mightyiam** [comment]
Thank you very much!

**Pull request merged**